dali/imgcodec/decoders/decoder_test_helper.h

```diff
@@ -76,6 +76,43 @@ static void AssertEqualSatNorm(const Tensor<CPUBackend> &img,
 }
 
 
+
+template <typename OutputType, typename RefType>
```

Comment:
I'd appreciate some doxygen here. It's not clear what "similar" means only by the name

Reply:
I can add something.

Reply:
Done.